src/Enterspeed.Source.UmbracoCms.V8/_content/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Views/seed.view.html

```diff
@@ -1,4 +1,13 @@
 ﻿<div ng-controller="SeedController as vm" class="seed-dashboard">
+
+    <umb-load-indicator ng-if="vm.loadingConfiguration">
+    </umb-load-indicator>
+
+    <div ng-if="!vm.runJobsOnServer" class="info-box">
+        <strong>Enterspeed jobs will not run on this current server.</strong><br />
+        Enterspeed jobs will only run on servers with Umbraco role of <i>Master</i> or <i>Single</i>. Current Umbraco server role is <i>{{vm.serverRole}}</i>.
```

Comment:
Should we add something about "Enterspeed.RunJobsOnAllServerRoles" as an option?